app/javascript/components/tpi/AscorRecentEmissions.js

```diff
@@ -1,8 +1,97 @@
 import PropTypes from 'prop-types';
+import Select from './Select';
+import React, {useState} from 'react';
 
-// eslint-disable-next-line no-empty-pattern
-const AscorRecentEmissions = ({ }) => {
+const AscorRecentEmissions = ({
+  emissions_metric_filter,
+  default_emissions_metric_filter,
+  emissions_boundary_filter,
+  default_emissions_boundary_filter,
+  trend_filters,
+  default_trend_filter,
+  data
+}) => {
+  const [filters, setFilters] = useState({
+    emissions_metric: default_emissions_metric_filter,
+    emissions_boundary: default_emissions_boundary_filter,
+    trends: default_trend_filter
+  });
+  const recentEmissions = data.filter((d) => d.emissions_metric === filters.emissions_metric && d.emissions_boundary === filters.emissions_boundary)[0] || {};
+  const trend = recentEmissions.trend || {};
+  const trendValue = trend.values?.filter((t) => t.filter === filters.trends)[0] || {};
 
+  const handleSelect = (opt) => {
+    setFilters({ ...filters, [opt.name]: opt.value });
+  };
+
+  return (
+    <>
+      <div className="emissions__filters assessments">
+        <div className="emissions__filters__emissions">
+          <Select
+            options={emissions_metric_filter}
+            onSelect={handleSelect}
+            value={filters.emissions_metric}
+            name="emissions_metric"
+            placeholder="Emissions Metric"
+            label="Emissions Metric"
+          />
+          <Select
+            options={emissions_boundary_filter}
+            onSelect={handleSelect}
+            value={filters.emissions_boundary}
+            name="emissions_boundary"
+            placeholder="Emissions Boundary"
+            label="Emissions Boundary"
+          />
+          <Select
+            options={trend_filters}
+            onSelect={handleSelect}
+            value={filters.trends}
+            name="trends"
+            placeholder="Trends"
+            label="Trends"
+          />
+        </div>
+      </div>
+      <div className="country-assessment__metric">
+        <div className="country-assessment__metric__title">
+          i. What is the country&apos;s most recent emissions level?
+        </div>
+        { recentEmissions.source && (
+          <div className="country-assessment__metric__source">
+            <a href={recentEmissions.source}>Source ({recentEmissions.year})</a>
```

Comment:
```suggestion
            <a target="_blank" rel="noreferrer" href={recentEmissions.source}>Source ({recentEmissions.year})</a>
```
I think it's better for the user experience to open the links in another tab